app/routes.py

```diff
@@ -1,2 +1,190 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from sqlalchemy import desc, asc
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
 
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
```

Comment:
To make this file shorter, we can separate the goal routes and task routes into different files.